packages/grid/x-data-grid-pro/src/tests/editComponents.DataGridPro.test.tsx

```diff
@@ -104,6 +104,7 @@ describe('<DataGridPro /> - Edit Components', () => {
       );
 
       const cell = getCell(0, 0);
+      fireEvent.mouseUp(cell);
```

Comment:
Is it necessary?
In `cellEditing.DataGridPro.test.tsx` you remove all the `cell.focus()` but do not add `fireEvent.mouseUp()`

Reply:
Yes, even though it's a double click, it uses the cell that received `mouseUp` to update the state. Below, `cellParams` is assigned by the `mouseUp` listener. If we don't fire it, the focus is never updated.

https://github.com/mui-org/material-ui-x/blob/9285a97a5e19eca7d06cb8f3d06e0ee1a1f05823/packages/grid/_modules_/grid/hooks/features/focus/useGridFocus.ts#L167-L168